PROJECT_PROGRESSION.md

```diff
@@ -16,14 +16,73 @@ This governance policy sets forth the proposal process for projects to be accept
 
 New projects should express interest to join the OpenJS Foundation via an email to new-projects@lists.openjsf.org with a filled out [Project Application Template](./NEW_PROJECT_APPLICATION.md). The initial application will have a silent period for internal only discussion. If your application is denied during this phase you will be notified privately. If there are no objections to the application, projects will given an `incubation` status and begin the on-boarding process. **Note** the `incubation` status is temporary as we determine if the project is a good fit. At any time during the `incubation` process a project can withdraw either voluntarily or at the request of the CPC.
 
-#### Application Phases
+#### Roles
+
+##### Applicant
+
+Individual or group who submits the application on behalf of a project to join the OpenJS Foundation.
+
+##### Application Manager
+
+The Application Manager ensures progress through the process, helps the project understand the work to be 
+completed along with explaining where they are in the application process. Applications Managers can be a
+member of the OpenJS Foundation staff, CPC Member, or other person approved by the CPC.
+
+##### Application Champion
+
+The Application Champion is a member of the CPC who commits to do the following:
```

Comment:
Can a member of the CPC be the same person sending the application?

Reply:
I don't think we've discussed one way or the other. My first thought is that there might be some conflict of interest but I'm not sure.  On the other hand it would make some aspects easier. I think we should discuss this outside of this PR which is going to land soon.  @mcollina can you open a new issue to discuss?

Reply:
Sure!